core/chains/evm/config/chain_scoped_gas_estimator.go

```diff
@@ -137,11 +137,11 @@ func (d *daOracleConfig) OracleAddress() *types.EIP55Address {
 }
 
 // CustomGasPriceCalldata returns the calldata for a custom gas price API.
-func (d *daOracleConfig) CustomGasPriceCalldata() string {
-	if d.c.OracleType == toml.DAOracleCustomCalldata {
+func (d *daOracleConfig) CustomGasPriceCalldata() *string {
+	if *d.c.OracleType == toml.DAOracleCustomCalldata {
```

Comment:
Is it OK to assume this is non-nil? If so, shouldn't the fallback.toml have a value?

Reply:
thanks for the catch, added a nil check